lessons/translating-words-math/langs/en-us/index.adoc

```diff
@@ -0,0 +1,176 @@
+
+| Key Points For The Facilitator
+|
+* This lesson may challenge students' ideas of math as a subject that is entirely black and white. This theme - that there are oftentimes a variety of completely valid ways of seeing a seemingly simple problem - will emerge again and again throughout the course.
+|===
+
+== Circles of Evaluation @duration{30 minutes}
+
+=== Overview
+Students match Circles of Evaluation to mathematical expressions, and then they consider how mathematical expressions in words map onto Circles of Evaluation.
+
+=== Launch
+
+Explain to students that they are going to receive @handout{pages/three-sets-cards.adoc, "three sets of cards"}.
+
+The first set of cards includes mathematical expressions. Students have definitely encountered those before! The expressions on their cards each include the number 15, the number 3, and an operator (+, -, *, or /).
+
+The second set of cards includes _Circles of Evaluation_. Students have almost definitely never encountered Circles of Evaluation before! Broadly, Circles of Evaluation will help students to visualize the structure of the mathematical expressions they encounter. For the time being, here’s what students need to know about Circles of Evaluation:
+
+- Every Circle must have one - and only one! - function, written at the top.
+
+- The inputs of the function are written left to right, in the middle of the Circle.
+
+Share these two key concepts before explaining the following instructions.
+
+[.lesson-instruction]
+- With your partner, match each A card with the corresponding B card.
+- Lay them out on the table in front of you so you can clearly see both the Circle and the expression. Create a separate pile for any cards that do not have a match.
+- Discuss each match that you make.
+
+Circulate as students sort their cards, ensuring that they are carefully analyzing each card so that they see the connection between the arithmetic expression and the Circle of evaluation. This first phase of matching cards should be relatively quick - but it might raise questions on a few topics:
+
+- *Symbols*. Circles utilize * to represent multiplication and / to represent division. (Why? Circles are a bridge representation - one which can eventually be used to help students learn to code! Although we use these symbols on all of our materials, you and your students can use whichever operator symbols are most comfortable.)
+
+- *Order of terms*. Although 5 + 3 and 3 + 5 both evaluate to 8, the Circle of Evaluation that corresponds with each of these expressions is unique.
+
+- *Position of the operator*. The operator always belongs at the top of the Circle, and not in between terms.
+
+[.lesson-instruction]
+- Now, you’re going to receive a set of C cards.
+- One at a time, take turns reading each C card out loud. After reading the card, place it with the appropriate Circle and expression.
```

Comment:
As the teacher, I want to know what range of things students might see on the C cards!